packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js

```diff
@@ -116,25 +119,28 @@ export default class QuanticSearchInterface extends LightningElement {
    * @param {SearchEngine} engine
    */
   initialize = (engine) => {
-    const {updateQuery} = CoveoHeadless.loadQueryActions(engine);
-
-    if (!this.disableStateInUrl) {
-      this.initUrlManager(engine);
-    }
+    if (!this.initialized) {
```

Comment:
```suggestion
    if (this.initialized) {
      return;
    }
```
Do we prefer this instead of a big if?